packages/blade/src/components/Dropdown/_decisions/decisions.md

```diff
@@ -0,0 +1,697 @@
+
+#### API
+
+Sample Usage
+
+```jsx
+<SelectInput icon={CalendarIcon} label="Your favorite Design System" helpText="Select only one" />
+```
+
+Props
+
+Mostly similar to other `Input` components extended from `BaseInput`
+
+```ts
+type SelectInputProps = {
+  // ... FormLabelProps like `label`, `labelPosition`
+  // ... FormValidationProps like `validationState`, `helpText`, `errorText`, etc.
+  // ... State Props like `isRequired`, `isDisabled`,
+  icon: IconComponent;
+};
+```
+
+#### A11y
+
+Would be ideal if we can render it as `button` with `aria-expanded` prop to handle state of dropdown open and close.
+
+Reference: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-expanded#buttons
+
+### `DropdownOverlay`
+
+An Overlay that positions dropdown correctly.
+
+Should handle the animations for dropdown.
+
+#### API
+
+Usage
+
+```jsx
+<Dropdown>
+  <SelectInput label="Click For Greetings" />
+  <DropdownOverlay>Hi</DropdownOverlay>
+</Dropdown>
+```
+
+Props
+
+_No Props Needed_
+
+<br/><br/>
+
+## ActionList
+
+Plain list that can be rendered inside dropdown, in navigation menu, or anywhere. It has selectable items that perform some action on click. It can also have multiple selectable items that render as checkboxes
+
+![](2022-12-08-16-58-46.png)
+
+### Design
+
+- [Figma - ActionList]()
+
+### API
+
+#### Simplest Usage Possible
+
+```jsx
+<ActionList
+  surfaceLevel={2}
+  onChange={(e) => {
+    console.log({ name: e.name, values: e.values });
+  }}
+>
+  <ActionListItem
+    title="Accept API"
+    value="accept"
+    desciption="Accept this API"
+    leading={<ActionListItemIcon icon={StarIcon} />}
+    trailing={<ActionListItemText>⌘ + Space + A</ActionListItemText>}
+    onClick={() => {}}
+  />
+  <ActionListItem
+    title="Delete"
+    desciption="Delete this value"
+    value="delete"
+    intent="negative"
+    leading={<ActionListItemIcon icon={TrashIcon} />}
+    trailing={<ActionListItemText>⌘ + Space + D</ActionListItemText>}
+    onClick={() => {}}
+  />
+</ActionList>
+```
+
+#### Most Complex Usage Possible
+
+```jsx
+<ActionList surfaceLevel={2}>
+  <ActionListHeader title="Recent Searches" icon={HistoryIcon} />
+  <ActionListSection title="Section Heading">
+    <ActionListItem
+      title="Accept API"
+      value="accept"
+      desciption="Accept this API"
+      leading={<ActionListItemIcon icon={StarIcon} />}
+      trailing={<ActionListItemText>⌘ + Space</ActionListItemText>}
+    />
+  </ActionListSection>
+  <ActionListItem
+    title="Delete"
+    desciption="Delete this value"
+    value="delete"
+    intent="negative"
+    leading={<ActionListItemIcon icon={TrashIcon} />}
+    trailing={<ActionListItemText>⌘ + Space + D</ActionListItemText>}
+  />
+  <ActionListFooter
+    title="Footer Title"
+    description="Footer Description"
+    leading={<ActionListFooterIcon icon={DocIcon} />}
+    trailing={
```

Comment:
So it can't be similar to Cards because in Cards we just have primary and secondary actions wheras in this footer, The footer in dropdown has too many variations. 

<img width="696" alt="image" src="https://user-images.githubusercontent.com/30949385/207027967-90b19fb4-8623-4f24-befe-95ba364ba786.png">

Supporting these variations with stricter API like that just makes the API really really complicated. I have added what the API would look like with props in Alternate Footer API in ActionListFooter section.

<img width="552" alt="image" src="https://user-images.githubusercontent.com/30949385/207028303-9dbb684e-71be-47d9-ae68-bf022352a260.png">